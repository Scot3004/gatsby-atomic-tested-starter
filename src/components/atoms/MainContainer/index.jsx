import styled from 'styled-components';

export const MainContainer = styled.main`
  grid-area: main;
  min-width: 0;
`;

export default MainContainer;
