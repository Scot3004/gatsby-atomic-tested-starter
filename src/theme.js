const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
