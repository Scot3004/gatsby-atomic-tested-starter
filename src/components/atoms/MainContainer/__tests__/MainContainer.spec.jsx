import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import MainContainer from '..';

describe('When I visit the site', () => {
  const tree = renderer.create(<MainContainer theme={theme} />).toJSON();
  test('Then I can see a wrapper without scrolls', () => {
    expect(tree).toMatchSnapshot();
  });
});
