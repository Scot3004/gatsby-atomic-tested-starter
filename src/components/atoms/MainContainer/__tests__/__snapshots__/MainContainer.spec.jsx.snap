// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I visit the site Then I can see a wrapper without scrolls 1`] = `
.c0 {
  grid-area: main;
  min-width: 0;
}

<main
  className="c0"
/>
`;
